youtube.js

```diff
@@ -81,24 +81,9 @@ $("body").unbind("click").on("click", "yt-live-chat-text-message-renderer,yt-liv
 
 });
 
-$("body").on("click", ".btn-clear", function () {
-  $(".hl-c-cont").addClass("fadeout").delay(300).queue(function(){
-    $(".hl-c-cont").remove().dequeue();
-  });
-});
 
 $( "yt-live-chat-app" ).before( '<highlight-chat></highlight-chat><button class="btn-clear">CLEAR</button>' );
 
-// Show a placeholder message so you can position the window before the chat is live
-$(function(){
-  var chatmessage = "this livestream is the best!";
-  var chatimg = "https://pin13.net/youtube-live-chat-sample-avatar.png";
-  $( "highlight-chat" ).addClass("preview").append('<div class="hl-c-cont fadeout"><div class="hl-name">Sample User<div class="hl-badges"></div></div><div class="hl-message">' + chatmessage + '</div><div class="hl-img"><img src="' + chatimg + '"></div></div>')
-  .delay(10).queue(function(next){
-    $( ".hl-c-cont" ).removeClass("fadeout");
-    next();
-  });
-});
 
 // Restore settings
```

Comment:
Most of the rest of this file can be deleted, except the mutation observer, which should probably be moved, and then turned into a function that takes the right CSS selectors.